neqo-transport/src/connection/mod.rs

```diff
@@ -88,6 +88,11 @@ pub use crate::send_stream::{RetransmissionPriority, SendStreamStats, Transmissi
 /// handshake.  This is a hack, but a useful one.
 const EXTRA_INITIALS: usize = 4;
 
+/// Maximum number of dummy datagrams to send before Client Initial packets.
+const MAX_PRE_INIT_PKTS: usize = 1;
+/// Payload size range (in bytes) for pre-Initial packets.
+const PRE_INIT_PAYLOAD_LEN_RANGE: (usize, usize) = (1200, 1400);
```

Comment:
I like the `noise` terminology you used; suggest to apply it more consistently, hence the rename here. Check if similar renamings make sense elsewhere?

```suggestion
const PRE_INIT_NOISE_LEN_RANGE: Range<usize> = (1200, 1400);
```
Is the 1200-1400 range chosen for any particular reason? If so, might want to add a comment.

Reply:
Would a smaller payload, e.g. 10 bytes, do as well?

Reply:
We chose that range to mimic actual Initial packet sizes. A smaller payload could work, but if it is less than the 1200-byte minimum defined in the RFC, it would be trivial to determine whether the first packet is a Client Initial.